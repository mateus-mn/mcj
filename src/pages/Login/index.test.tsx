import { render } from '@testing-library/react';
import { QueryClient, QueryClientProvider } from 'react-query';
import LoginPage from '.';

describe('Testes da pÃ¡gina de login', () => {
	it('Teste de snapshot', () => {
		const queryClient = new QueryClient();
		const { container } = render(
			<QueryClientProvider client={queryClient}>
				<LoginPage />
			</QueryClientProvider>,
		);

		expect(container).toMatchSnapshot();
	});
});
