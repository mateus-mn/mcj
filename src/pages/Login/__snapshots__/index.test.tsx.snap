// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testes da p√°gina de login Teste de snapshot 1`] = `
<div>
  <div
    class="container mt-3"
  >
    <div
      class="row"
    >
      <div
        class="col-11 offset-1"
      >
        <h1>
          Login
          <img
            alt="Logo do MCJ"
            src="logoMcj.png"
            width="50"
          />
        </h1>
      </div>
    </div>
    <form
      novalidate=""
    >
      <div
        class="row mt-3"
      >
        <div
          class="col-5 offset-1"
        >
          <div
            class="form-group"
          >
            <label
              for="login"
            >
              Login 
              <svg
                aria-hidden="true"
                class="svg-inline--fa fa-asterisk "
                data-icon="asterisk"
                data-prefix="fas"
                focusable="false"
                role="img"
                viewBox="0 0 384 512"
                xmlns="http://www.w3.org/2000/svg"
              >
                <path
                  d="M192 32c17.7 0 32 14.3 32 32V199.5l111.5-66.9c15.2-9.1 34.8-4.2 43.9 11s4.2 34.8-11 43.9L254.2 256l114.3 68.6c15.2 9.1 20.1 28.7 11 43.9s-28.7 20.1-43.9 11L224 312.5V448c0 17.7-14.3 32-32 32s-32-14.3-32-32V312.5L48.5 379.4c-15.2 9.1-34.8 4.2-43.9-11s-4.2-34.8 11-43.9L129.8 256 15.5 187.4c-15.2-9.1-20.1-28.7-11-43.9s28.7-20.1 43.9-11L160 199.5V64c0-17.7 14.3-32 32-32z"
                  fill="currentColor"
                />
              </svg>
            </label>
            <input
              class="form-control"
              id="login"
              name="login"
            />
          </div>
        </div>
        <div
          class="col-5"
        >
          <div
            class="form-group"
          >
            <label
              for="senha"
            >
              Senha 
              <svg
                aria-hidden="true"
                class="svg-inline--fa fa-asterisk "
                data-icon="asterisk"
                data-prefix="fas"
                focusable="false"
                role="img"
                viewBox="0 0 384 512"
                xmlns="http://www.w3.org/2000/svg"
              >
                <path
                  d="M192 32c17.7 0 32 14.3 32 32V199.5l111.5-66.9c15.2-9.1 34.8-4.2 43.9 11s4.2 34.8-11 43.9L254.2 256l114.3 68.6c15.2 9.1 20.1 28.7 11 43.9s-28.7 20.1-43.9 11L224 312.5V448c0 17.7-14.3 32-32 32s-32-14.3-32-32V312.5L48.5 379.4c-15.2 9.1-34.8 4.2-43.9-11s-4.2-34.8 11-43.9L129.8 256 15.5 187.4c-15.2-9.1-20.1-28.7-11-43.9s28.7-20.1 43.9-11L160 199.5V64c0-17.7 14.3-32 32-32z"
                  fill="currentColor"
                />
              </svg>
            </label>
            <input
              class="form-control"
              id="senha"
              name="senha"
              type="password"
            />
          </div>
        </div>
      </div>
      <div
        class="row mt-3"
      >
        <div
          class="col-11 offset-1"
        >
          <button
            class="btn btn-success mt-3"
            type="submit"
          >
            Acessar
          </button>
        </div>
      </div>
    </form>
  </div>
</div>
`;
